src/js/components/MainPanel/mainPanel.spec.jsx

```diff
@@ -27,8 +22,8 @@ const getWrapper = () => mount(
 );
 
 describe('A suite', () => {
```

Comment:
How about making it a little bit more descriptive?
Or is it a common practice in this project? @SzajekSzymon @mikolajkozakiewicz